recipes/mimalloc/all/patches/0003-get-rid-of-mi_install_dir.patch

```diff
@@ -0,0 +1,91 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 71772d8..6fe421a 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -206,7 +206,7 @@ endif()
+ # Install and output names
+ # -----------------------------------------------------------------------------
+ 
+-set(mi_install_dir "${CMAKE_INSTALL_PREFIX}/lib/mimalloc-${mi_version}")
++# set(mi_install_dir "${CMAKE_INSTALL_PREFIX}/lib/mimalloc-${mi_version}")
```

Comment:
It might be a bit easier to change the value of mi_install_dir to some default location and decrease the patch size below this line, if it's possible.

Reply:
Fixed in 3c836906